### ISSUE TITLE
Load all pages at once in memory

### ISSUE THREAD

**Issue by ShivamArora** [CrossPT/flutter_plugin_pdf_viewer#19]
I used `PDFDocument.fromURL(url)` to load the document and view using `PDFViewer(document:document)`.

Now, when I try to move to another page, it loads the next/previous page asynchronously every time I move to a new page and shows a `CircularProgressIndicator` for that period.

To provide a better user experience, you should load all the pages at once or load 3-4 pages next and previous to the current page in advance, so the user doesn't have to wait while moving between pages.

**Comment by Levi-Lesches** [comment]
I agree with this 100%. @CrossPT, and plans for this? I'd be happy to help if needed.

**Comment by calvinsug** [comment]
100% agree for sure, it will be painful for me to read 9 pages and i have to wait for a several seconds to go to the next page
